Chemical-starting-points.md

```diff
@@ -0,0 +1,45 @@
+# Appropriate starting points for applying a SMIRNOFF
+
+SMIRNOFF force fields use direct chemical perception meaning that, unlike many molecular mechanics (MM) force fields, they apply parameters based on substructure searches acting directly on molecules.
+This creates unique opportunities and allows them to encode a great deal of chemistry quite simply, but it also means that the *starting point* for parameter assignment must be well-defined chemically, giving not just the elements and connectivity for all of the atoms of all of the components of your system, but also providing the formal charges and bond orders.
+
+Specifically, to apply SMIRNOFF to a system, you must either:
+1. Provide molecules (currently `OEMol` objects, but an RDKit version is in development) corresponding to the components of your system, or
+2. Provide an OpenMM `Topology` which includes bond orders and thus can be converted to molecules corresponding to the components of your system
+
+Without this information, identifying the chemical identity of the molecules comprising your system becomes a matter of guesswork rather than certainty.
+Different molecules can have the same chemical graph but differ in bond order and formal charge, or different resonance structures may be treated rather differently by some force fields (e.g. `c1cc(ccc1c2cc[nH+]cc2)[O-]` vs `C1=CC(C=CC1=C2C=CNC=C2)=O`, where the central bond is rotatable in one resonance structure but not in the other) even though they have identical formal charge and connectivity (chemical graph).
+A force field which uses the chemical identity of molecules to assign parameters needs to know what those molecules are, first.
```

Comment:
I don't understand what you're trying to say here.

Reply:
Can you be more specific? I'm trying to say that if you don't have bond order information and formal charges, you don't actually know what molecule you are talking about in some cases, and in other cases, you don't know what resonance structure you are trying to set up, etc. And you can't assign parameters until you've specified that.

Which part seems most confusing?